cmd/registry/controller/controller.go

```diff
@@ -26,7 +26,7 @@ import (
 type Action struct {
 	Command           string
 	GeneratedResource string
-	Placeholder       bool
+	Receipt           bool
```

Comment:
Does this mean that this action requires a receipt? (if so it might be clearer to name this "RequiresReceipt")

Reply:
Updated this to `RequiresReceipt`, makes the code more readable. Thanks!